### ISSUE TITLE
Cutting Board item Dupe

### ISSUE THREAD

**Issue by TarConvent** [Tismas/cave-factory#16]
When placing a cutting board in the world, picking up the block renders air and gives you the item. Alobg with setting the air down it renders the cutting boad making the duping possible

**Comment by BrockDierksen** [comment]
Yes, Carry On, does do this. I used it in a pack a while back and noticed this mod can cause many issues. If you ever need to shift right click a block you have to get as far as you can to be able to.

**Comment by Kylecoler** [comment]
When the board is in my hand and I place it all is fine. After I use shift right click to pick up the board, 1. the board dupes giving me a visible item and a blank item in the hot-bar slot I used to pickup the board, 2. At that point i can not change which hot-bar slot I have selected and where ever I right click I place the invisible board on the most available surface. 3. after the board is placed I have control of my hot-bar again. breaking the board instead of shift right clicking seems to be just fine for me.
I did open up the game output and did not see any entries in relation to this dupe glitch.
